tests/cases/helpers/case_helpers_care_platform.yaml

```diff
@@ -1714,4 +1714,29 @@ CarePartnerEditParticipantNavigation:
       Strategy: xpath
       Id: $PAGE.care_platform_participant_records.edit_info_button$
     - Type: sleep
-      Time: 1
\ No newline at end of file
+      Time: 1
+
+CarePartnerToastValidation:
+  Environment: Settings
+  Roles:
+    - Role: desktopChrome
+      App: desktop
+  Actions:
+    - Type: wait_for
+      Strategy: xpath
+      Id: $PAGE.care_platform_notifications.add_vendor_notifications$
```

Comment:
The notification element should not be call ad_vendor as you have defined it as a general element for all notifications on the carepartner platform. change that element to be something like carepartner_toast_notificaiton and place it on the care_platform general page

Reply:
I did it in this commit https://github.com/neveralone-chs-org/TestRay/pull/156/commits/46fdcf3172b820c39c96da86ac61959003acf08c